specify/specify7
# Display create RS and Gallery tool only for factious RS with same type

Fixes #4898

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Go to data entry (ex CO)
- save form
- [ ] verify that Create RS tool are not available 
-  create a new data entry form using the same table as in first step (ex CO)
- [ ] verify that Gallery icon and Create RS tool are not available 
- save form 
- [ ] verify that Gallery icon and Create RS tool are available 
- create a new data entry form using another table than in first step (ex CE)
- [ ] verify that Gallery icon and Create RS tool are not available 
- save 
- [ ] verify that Gallery icon and Create RS tool are not available
